cg_hermes/config/balsamic.py

```diff
@@ -11,6 +11,13 @@
     "BALSAMIC_X.X.X_graph.pdf": ["balsamic-dag"],
     "multiqc_report.html": ["html", "multiqc-html"],
     "multiqc_data.json": ["json", "multiqc-json"],
+    "multiqc_picard_AlignmentSummaryMetrics.json": ["multiqc", "picard", "alignmentsummarymetrics"],
+    "multiqc_picard_dups.json": ["multiqc", "picard", "dups"],
+    "multiqc_picard_HsMetrics.json": ["multiqc", "picard", "hsmetrics"],
+    "multiqc_picard_insertSize.json": ["multiqc", "picard", "insertsize"],
+    "multiqc_picard_wgsmetrics.json": ["multiqc", "picard", "wgsmetrics"],
+    "multiqc_samtools_stats.json": ["multiqc", "samtools", "stats"],
+    "multiqc_somalier.json": ["multiqc", "somalier"],
```

Comment:
Nice I think these tags offer a way to pull out these files from HK easily for a given bunde.